tools/lib/h5tools_utils.c

```diff
@@ -1079,19 +1079,19 @@ h5tools_parse_ros3_fapl_tuple(const char *tuple_str, int delim, H5FD_ros3_fapl_e
  *     {   aws_region,
  *         secret_id,
  *         secret_key,
+ *         session_token (optional)
  *     }
- *     If all three strings are empty (""), the default fapl will be default.
- *     Both aws_region and secret_id values must be both empty or both
- *         populated. If
- *     Only secret_key is allowed to be empty (the empty string, "").
+ *     If all three (or four) strings are empty (""), the default fapl will be
+ *     default. Both aws_region and secret_id values must be both empty or both
```

Comment:
"Both" at the beginning of the sentence seems redundant;  not terribly important, though.